TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -933,8 +921,7 @@ public class RoutingManager
                                         (nextIncomingDir == ArrowDirection.Right && hasLeftArrow) || // valid incoming right
                                         (nextIncomingDir == ArrowDirection.Left && hasRightArrow) || // valid incoming left
                                         (nextIncomingDir == ArrowDirection.Forward && hasForwardArrow) || // valid incoming straight
-                                        (nextIncomingDir == ArrowDirection.Turn && canTurn) /*valid turning lane*/) 
-                                    {
+                                        (nextIncomingDir == ArrowDirection.Turn && canTurn) /*valid turning lane*/) {
```

Comment:
auto formatting has done this when I added the final bracket. ignore if there is only white space changes.